public/js/calendar.js

```diff
@@ -1,3 +1,121 @@
+function generateWebCalendar(data, calendarEl, room) {
+  const calendar = new FullCalendar.Calendar(calendarEl, {
+    plugins: ['interaction', 'timeGrid', 'dayGrid'],
+    header: {
+      left: 'prev,next today',
+      center: 'title',
+      right: 'timeGridOneDay,timeGridWeek,dayGridMonth'
+    },
+    defaultView: 'timeGridWeek',
+    views: {
+      timeGridOneDay: {
+        type: 'timeGrid',
+        duration: { days: 1 },
+        buttonText: 'nap',
+        slotLabelFormat: { 
+          hour: 'numeric',
+          minute: '2-digit',
+          omitZeroMinute: false
+        },
+        nowIndicator: true
+      }
+    },
+    buttonText: {
+      today: 'ma',
+      month: 'hónap',
+      week: 'hét'
+    },
+    titleFormat: {
+      year: 'numeric',
+      month: 'short',
+      weekday: 'short',
+      day: 'numeric'
+    },
+    locale: 'hu',
+    selectable: true,
+    events: data,
+    select: info =>
+      location.href = `/groups/new?start=${info.startStr}&end=${info.endStr}&roomId=${room}`
+    ,
+    eventClick: calEvent =>
+      location.href = `/groups/${calEvent.event.groupId}`
+  })
+  return calendar
+}
+
+function generateMobileCalendar(data, calendarEl, room) {
+  const calendar = new FullCalendar.Calendar(calendarEl, {
+    plugins: ['interaction', 'timeGrid', 'dayGrid'],
+    header: {
+      left: 'title',
+      center: '',
+      right: 'timeGridOneDay,dayGridMonth'
+    },
+    defaultView: 'timeGridOneDay',
+    views: {
+      timeGridOneDay: {
+        type: 'timeGrid',
+        duration: { days: 1 },
+        buttonText: 'nap',
+        slotLabelFormat: { 
+          hour: 'numeric',
+          minute: '2-digit',
+          omitZeroMinute: false
+        },
+        nowIndicator: true
+      }
+    },
+    buttonText: {
+      today: 'ma',
+      month: 'hónap',
+      week: 'hét'
+    },
+    customButtons: {
+      prevWeek: {
+        text: '-7',
+        click: () => {
+          calendar.incrementDate({
+            days: 7
+          })
+        }
+      },
+      nextWeek: {
+        text: '+7',
+        click: () => {
+          calendar.incrementDate({
+            days: -7
+          })
+        }
+      }
```

Comment:
The +7 and -7 are flipped.

Reply:
Fixed it. Now if I want to push the amended commit, do I have to force the push? (I've had bad experiences with forcing git commands, don't know if it's elegant enough)

Reply:
Yes, since the commit has has changed, you have to force push it. It's ok to force push to feature branches, since 99% of the time only one person will be working on it. I'd advise to use `--force-with-lease` option, it's much safer.

Reply:
Something seems to be wrong right now. :( I get this message when I push (tried amend and force, now tried pushing a simple commit):
```
remote: error: GH006: Protected branch update failed for refs/heads/ref/fullcalendar.
remote: error: Required status check "build (12.x)" is expected.
To https://github.com/kir-dev/tanulo-next.git
 ! [remote rejected] ref/fullcalendar -> ref/fullcalendar (protected branch hook declined)
```

Reply:
I temporary disabled the check, try now.